pimd/pim_iface.c

```diff
@@ -1674,7 +1674,9 @@ static int pim_ifp_up(struct interface *ifp)
 						__func__, vrf->name);
 					return 0;
 				}
+
 				pim_zebra_interface_set_master(master, ifp);
+				break;
```

Comment:
Will this still work for all the VRFs? I see this loop is trying to set the pimreg for all the VRFs but adding the break will introduce problem for VRFs, right?

Reply:
One interface can join only one master/table, so needn't walk all tables. Just skipped if matching one table.